tracee-ebpf/tracee/co-re/struct_flavors.h

```diff
@@ -3,30 +3,68 @@
 
 #pragma clang attribute push (__attribute__((preserve_access_index)), apply_to = record)
 
+#include <vmlinux.h> // linting purposes
```

Comment:
Why do we neeed vmlinux.h here?
Doesn't vmlinux.h includes this header?

Reply:
we dont, it was for my linting to stop complaining on errors. removing.